src/library/Uno.Themes.WinUI.Markup/Themes/Thumb.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class Thumb
+		{
+			public static class Resources
+			{
+				public static class Default
+				{
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "SliderThumbBackground")]
+						public static ResourceValue<Brush> Default => new("SliderThumbBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "SliderThumbBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("SliderThumbBackgroundDisabled", true);
+					}
+				}
+			}
+
+			public static class Styles
+			{
+				[ResourceKeyDefinition(typeof(Style), "MaterialSliderThumbStyle", TargetType = typeof(Microsoft.UI.Xaml.Controls.Primitives.Thumb))]
+				public static ResourceValue<Style> Default => new("MaterialSliderThumbStyle");
+			}
+		}
```

Comment:
Let's just move this to the Slider class as a subclass.

So `Slider.Resources.Thumb.Background.Default/Disabled` and `Slider.Styles.Thumb.Default`

Reply:
Done